### ISSUE TITLE
BDD100k Test Dataset

### ISSUE THREAD

**Issue by Abdul-Nasir11** [sunggukcha/deeplabs#9]
Hello Amazing Work !

As far as i know, in publicly available bdd100k dataset annotated masks of only training images (70k) and validation images (10k) are provided. How did you split the data into training, validation and testing datasets. In training code I found that there are separate folders for each of them.
Thanks in advance !

**Comment by sunggukcha** [comment]
Hi @Abdul-Nasir11 !

I used as the dataset is. 70k training images for training, 10k validation images for validation.
As far as I remember, I did not refactored the dataset and used as it is.

Thanks for your attention.

**Comment by Abdul-Nasir11** [comment]
Thanks a lot for your prompt reply !
what about the test result your obtained?  I mean which part of the dataset is used for testing?

**Comment by sunggukcha** [comment]
The test result is obtained from **test set**. I could get the result **from the official bdd site** that day, but I am not sure if it is still on.

**Comment by Abdul-Nasir11** [comment]
Got it ! test dataset is currently not available on the official website, I am in search of finding it but no luck so far.
Thanks  a lot !
